pkgs/top-level/stage.nix

```diff
@@ -87,6 +94,8 @@ let
   stdenvBootstappingAndPlatforms = self: super: {
     buildPackages = (if buildPackages == null then self else buildPackages)
       // { recurseForDerivations = false; };
+    __targetPackages = (if __targetPackages == null then self else __targetPackages)
```

Comment:
https://github.com/NixOS/nixpkgs/pull/25099/files#diff-69df24a05ec3d577cb638113e9837ae1R52 is a somewhat poorly-worded explanation of that.